yb-voyager/cmd/import.go

```diff
@@ -224,33 +224,31 @@ func registerImportDataFlags(cmd *cobra.Command) {
 			"Note: use-public-ip flag will be ignored if this is used.")
 	// flag existence depends on fix of this gh issue: https://github.com/yugabyte/yugabyte-db/issues/12464
 	BoolVar(cmd.Flags(), &tconf.DisableTransactionalWrites, "disable-transactional-writes", false,
-		"true - to disable transactional writes in tables for faster data ingestion (default false)\n"+
+		"Disable transactional writes in tables for faster data ingestion (default false)\n"+
 			"(Note: this is a interim flag until the issues related to 'yb_disable_transactional_writes' session variable are fixed. Refer: https://github.com/yugabyte/yugabyte-db/issues/12464)")
 	// Hidden for beta2.0 release (and onwards until further notice).
 	cmd.Flags().MarkHidden("disable-transactional-writes")
 
 	BoolVar(cmd.Flags(), &truncateSplits, "truncate-splits", true,
-		"true - to truncate splits after importing\n"+
-			"false - to not truncate splits after importing (required for debugging)")
+		"Truncate splits after importing")
 	cmd.Flags().MarkHidden("truncate-splits")
 }
 
 func registerImportSchemaFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&tconf.ImportObjects, "object-list", "",
-		"list of schema object types to include while importing schema")
+		"comma separated list of schema object types to include while importing schema")
```

Comment:
Also mention whether the names are case-sensitive?

Reply:
Asked @ShivanshGahlot  to take care of this as part of https://github.com/yugabyte/yb-voyager/pull/1147/files